content/kubelb/v1.0/references/crds/_index.en.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "CRD References"
+date = 2024-03-06T12:00:00+02:00
+weight = 40
++++
+
+This section contains a reference of the Kubermatic KubeLB Custom Resource Definitions.
```

Comment:
Does it make sense to have a lower level CRD References section under the References section whilst both say that `This section contains a reference of the Kubermatic KubeLB Custom Resource Definitions. `

Reply:
Good point, updated 👍